pkg/config/env_var_test.go

```diff
@@ -192,29 +199,68 @@ func TestRemoveEnvVarsFromList(t *testing.T) {
 			},
 		},
 		{
+			name: "Case 4: Check failing with foo=bar (invalid variable.. should error out)",
+			envVarList: EnvVarList{
+				EnvVar{
+					Name:  "foo",
+					Value: "bar",
+				},
+				EnvVar{
+					Name:  "foo",
+					Value: "bar",
+				},
+			},
+			expected: EnvVarList{},
+			keys:     []string{"foo=bar", "hi=hello"},
+			wantErr:  true,
+		},
+		{
+			name: "Case 5: Check failing when passing in multiple vals but one is valid",
 			envVarList: EnvVarList{
 				EnvVar{
 					Name:  "foo",
 					Value: "bar",
 				},
+				EnvVar{
+					Name:  "hi",
+					Value: "hello",
+				},
+			},
+			expected: EnvVarList{},
+			keys:     []string{"foo=bar", "hi"},
+			wantErr:  true,
+		},
+		{
+			name: "Case 6: Check failing when passing in nothing",
+			envVarList: EnvVarList{
 				EnvVar{
 					Name:  "foo",
 					Value: "bar",
 				},
+				EnvVar{
+					Name:  "hi",
+					Value: "hello",
+				},
 			},
 			expected: EnvVarList{},
-			keys:     []string{"foo"},
+			keys:     []string{""},
+			wantErr:  true,
```

Comment:
Why are we expecting an error here? Isn't this similar to case 3 where we don't pass any keys?

Reply:
We want an error because there are no keys passed

Reply:
But this looks similar to Case 3 where we're not having `keys` defined and we don't expect an error there.